.gitignore

```diff
@@ -1,3 +1,4 @@
 *~
 *.swp
 node_modules
+dist
```

Comment:
I'm in favor of adding dist in git, because it allows us to npm install from github.
What do you think?

Reply:
Up to you, the reason why I don't do it is because it bloats the git repositories.

Reply:
That's understandable, and it's probably the standard.
Let me keep it until I change my mind. 😄